src/ui/components/Timeline/ProgressBars.tsx

```diff
@@ -1,35 +1,27 @@
 import clamp from "lodash/clamp";
-import React from "react";
 
-import {
-  getCurrentTime,
-  getHoverTime,
-  getPlaybackPrecachedTime,
-  getZoomRegion,
-} from "ui/reducers/timeline";
+import { getCurrentTime, getHoverTime, getZoomRegion } from "ui/reducers/timeline";
 import { useAppSelector } from "ui/setup/hooks";
 import { getVisiblePosition } from "ui/utils/timeline";
 
 export default function ProgressBars() {
   const currentTime = useAppSelector(getCurrentTime);
   const hoverTime = useAppSelector(getHoverTime);
-  const precachedTime = useAppSelector(getPlaybackPrecachedTime);
   const zoomRegion = useAppSelector(getZoomRegion);
 
   const percent = getVisiblePosition({ time: currentTime, zoom: zoomRegion }) * 100;
   const hoverPercent = getVisiblePosition({ time: hoverTime, zoom: zoomRegion }) * 100;
-  const precachedPercent = getVisiblePosition({ time: precachedTime, zoom: zoomRegion }) * 100;
 
   return (
     <>
       <div className="progress-line full" />
       <div
         className="progress-line preview-max"
-        style={{ width: `${clamp(Math.max(hoverPercent, precachedPercent), 0, 100)}%` }}
```

Comment:
I'm not sure what this "pre-cached percent" state was. I don't _think_ we really need it. Particularly since it's only used for our internal expanded/enhanced timeline UI. So I removed it to simplify things a bit.